src/app/loans/loans-view/view-charge/view-charge.component.ts

```diff
@@ -52,7 +52,7 @@ export class ViewChargeComponent {
               private settingsService: SettingsService) {
     this.route.data.subscribe((data: { loansAccountCharge: any, loanDetailsData: any }) => {
       this.chargeData = data.loansAccountCharge;
-      this.allowPayCharge = (this.chargeData.chargeTimeType.code === 'chargeTimeType.instalmentFee');
+      this.allowPayCharge = this.chargeData.chargePayable;
```

Comment:
I am not familiar with this "chargePayable". Can you help me to learn more about this?

Reply:
It's related to the Charge Payment Method

<img width="1472" alt="Screenshot 2023-01-03 at 13 53 29" src="https://user-images.githubusercontent.com/44206706/210601463-46fc307e-996b-4f8f-a9ad-0cb1b1c89235.png">

In the Charge definition must be set as Account Transfer
<img width="899" alt="Screenshot 2023-01-03 at 13 56 17" src="https://user-images.githubusercontent.com/44206706/210601601-ede0f3de-4ef1-453a-9135-0cf7016f1d1f.png">